UI/window-basic-main-outputs.cpp

```diff
@@ -393,6 +393,9 @@ void SimpleOutput::Update()
 			"UseAdvanced");
 	bool enforceBitrate = config_get_bool(main->Config(), "SimpleOutput",
 			"EnforceBitrate");
+	bool dynamicBitrate = config_get_bool(main->Config(), "SimpleOutput",
+		"DynamicBitrate");
```

Comment:
Style:  Tab twice to indent a line continuation.

Reply:
done

Reply:
Sorry.  I meant tab twice from the start of the previous line, like 394 and 395 above.